package.json

```diff
@@ -59,8 +59,8 @@
     "author": "reinierguerra",
     "license": "",
     "engines": {
-        "node": "18.x",
-        "yarn": "1.22.19"
+        "node": ">=18.x",
```

Comment:
We could change this to more recent versions. But not sure if it would break anything. For now, simply adding a >=.